### PR TITLE
cli: status and list with start/finished times

### LINKED ISSUE #370: cli: `list` and `status` to inform about start/stop times
**Current behaviour**

The `list` and `status` commands return only the workflow creation date:

```
$ reana-client list
NAME                  RUN_NUMBER   CREATED               STATUS
root6-roofit-yadage   1            2020-02-07T10:54:59   finished
root6-roofit-serial   1            2020-02-07T10:54:15   finished
root6-roofit-cwl      1            2020-02-07T10:53:36   finished
$ reana-client status -w root6-roofit-cwl
NAME               RUN_NUMBER   CREATED               STATUS     PROGRESS
root6-roofit-cwl   1            2020-02-07T10:53:36   finished   2/2
```

This is OK for queued workflows, but if a workflow started or if it has finished, it'll be useful to expose start up time and finished time as well.

**Expected behaviour**

The `status` command should definitely return start/stop times: 

```
$ reana-client status -w root6-roofit-cwl
NAME               RUN_NUMBER   CREATED               STARTED              STOPPED               STATUS     PROGRESS 
root6-roofit-cwl   1            2020-02-07T10:53:36   2020-02-07T10:53:36  2020-02-07T10:53:36   finished   2/2
```

The `list` command could do as well since it is not too expensive (=the same DB lookup).

We could call the new output columns STARTED (this one is easy) and STOPPED (this one is harder, "stopped" would be the mirror-word to "started", but it may not be the best, as its meaning can be misinterpreted).  The "finished/completed/etc" column name should fit both "finished" and "failed" workflows, so we should find something generic.  Could even use "begun/ended" or somehing... Muse more IRL when this task is taken up.

### PR DESCRIPTION
**Pull request by diegodelemos**
* Adds new columns to report workflow start and finished times
  (closes #370).

### CODE REVIEW

**Comment by diegodelemos** [comment]
Depends on https://github.com/reanahub/reana-server/pull/212, https://github.com/reanahub/reana-workflow-controller/pull/294 and https://github.com/reanahub/reana-commons/pull/166.

File: reana_client/cli/workflow.py [hunk 1]
```diff
@@ -129,7 +129,8 @@ def workflow_workflows(ctx, sessions, _filter, output_format, access_token,
         response = get_workflows(access_token, type, bool(verbose), block_size)
         verbose_headers = ['id', 'user', 'size']
         headers = {
-            'batch': ['name', 'run_number', 'created', 'status'],
+            'batch': ['name', 'run_number', 'created', 'run_started_at',
```
Comment (roksys) [inline reana_client/cli/workflow.py]: I think @tiborsimko wanted to call these columns `started` and `completed` see - https://github.com/reanahub/reana-client/issues/370#issue-561578866

Comment (tiborsimko) [reply reana_client/cli/workflow.py]: Can you bump up the devYYYYMMDD version number as part of the commit?

**Comment by diegodelemos** [comment]
Rebased and comments (https://github.com/reanahub/reana-client/issues/370#issuecomment-584370292) taken into account. CC @tiborsimko

**Pull request merged**